lib/android-helpers.js

```diff
@@ -135,14 +156,7 @@ helpers.getDeviceInfoFromCaps = async function (opts = {}) {
   // on instantiating on earlier (at this point, we have no udid)
   // we can only use this ADB object for commands that would not be confused
   // if multiple devices are connected
-  const adb = await ADB.createADB({
-    javaVersion: opts.javaVersion,
-    adbPort: opts.adbPort,
-    remoteAdbHost: opts.remoteAdbHost,
-    suppressKillServer: opts.suppressKillServer,
-    clearDeviceLogsOnStart: opts.clearDeviceLogsOnStart,
-    adbExecTimeout: opts.adbExecTimeout,
-  });
+  const adb = await helpers.createBaseADB(opts);
   let udid = opts.udid;
```

Comment:
can we also assign these local vars in one-liner, like it's done below?

Reply:
It is one line. The `emPort` should not be initialized from the `opts`.